### ISSUE TITLE
RainLoop Webmail does not work on ownCloud 8

### ISSUE THREAD

**Issue by jochen35** [RainLoop/rainloop-webmail#439]
RainLoop Webmail does not work on ownCloud 8.0 (stable). The webmail URL and the path will not be saved. The entries rainloop-path and rainloop-url missing in the DB (oc_appconfig).

![owncloud_1](https://cloud.githubusercontent.com/assets/6515937/6109134/07065baa-b079-11e4-9138-6c3cbc36358f.png)

**Comment by RainLoop** [comment]
Interesting, any errors in any logs?

**Comment by jelle-e** [comment]
Same here. No errors in logs anywhere :S

**Comment by jochen35** [comment]
I have the logging of Rainloop active, but no messages are generated. Also in ownCloud log there are no entries. A click on the icon Rainloop of ownCloud opens the view of all files.

**Comment by buzz** [comment]
Exactly the same here since the upgrade to OC 8.0...

**Comment by RainLoop** [comment]
Ok.

Try to create file <owncload>/apps/rainloop/appinfo/routes.php

```php
<?php

$this->create('rainloop_ajax_index', '/')->actionInclude('rainloop/index.php');
$this->create('rainloop_ajax_personal', 'ajax/personal.php')->actionInclude('rainloop/ajax/personal.php');
$this->create('rainloop_ajax_admin', 'ajax/admin.php')->actionInclude('rainloop/ajax/admin.php');
```

If it helps, write here, I will add it to the package.

**Comment by jelle-e** [comment]
That fixed it! Thanks.

**Comment by RainLoop** [comment]
Good

**Comment by buzz** [comment]
Can confirm it works now.
Thank you!

**Comment by heathen711** [comment]
Using the above it will now save and load the app but it does not auto login with the saved credentials. Nothing in the error logs. Any other suggestions?

**Comment by f-s** [comment]
Hi, yep the auto-login is broken in my install too. Too bad ...

**Comment by buzz** [comment]
Auto-login works for me: ownCloud 8.0/RainLoop 1.8.0.251

**Comment by RainLoop** [comment]
Hi all,
You can try a new fixed version (beta) #452
